lib/common.js

```diff
@@ -412,6 +412,39 @@ exports.createTableForProjects = function(projects) {
   return table;
 };
 
+exports.createTableForTemplates = function(templates, type) {
+  var table;
+  switch (type) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASwitchWithoutDefault)